tests/e2e/api-core-tests/README.md

```diff
@@ -0,0 +1,133 @@
+# WooCommerce Core API Test Suite
+
+This package contains automated API tests for WooCommerce.
+
+## Environment variables
+
+Before running the tests, the following environment variables need to be configured as shown in `.env.example`:
```

Comment:
Can we include the alternative method of supplying these via the command line?

```
$ BASE_URL=http://localhost:8084 USERNAME=admin PASSWORD=password npm run test:api

Reply:
Good idea! Added.

Reply:
`USERNAME` and `PASSWORD` didn't work but the following does.

```
$ BASE_URL=http://localhost:8084 USER_KEY=admin USER_SECRET=password npm run test:api
```

Reply:
Noticed that too. Yesterday when I was testing the command line a bit more, and noticed the `process.env` ended up pulling in the system user, so I went and updated all of the `USERNAME` and `PASSWORD` values to be `USER_KEY` and `USER_SECRET` instead.